templates/4/header/header_welcomeblock_member.html

```diff
@@ -43,7 +43,7 @@
 			<div class="items">
 				<!-- Extra account avatars will be here, with links to switch thems -->
 				<a href="" title="Dodaj konto"><i class="cp cp-add"></i></a>
-				<a href="{$mybb->settings['bburl']}/logout.php" title="Wyloguj się"><i class="cp cp-power"></i></a>
+				<a href="member.php?action=logout&amp;logoutkey={$mybb->user['logoutkey']}" title="Wyloguj się"><i class="cp cp-power"></i></a>
```

Comment:
Usually mybb uses full links, so it should be like:
```suggestion
				<a href="{$mybb->settings['bburl']}/member.php?action=logout&amp;logoutkey={$mybb->user['logoutkey']}" title="Wyloguj się"><i class="cp cp-power"></i></a>
```

While what you used SHOULD work, mybb seems to always refer by full URL. I don't know why, but it would probably be safer if we never had to learn why...